app/src/main/java/org/oppia/app/profile/ProfileChooserViewModel.kt

```diff
@@ -36,6 +38,12 @@ class ProfileChooserViewModel @Inject constructor(
       ProfileChooserUiModel.newBuilder().setProfile(it).build()
     }.toMutableList()
 
+    profileList.forEach {
+      if (it.profile.avatar.avatarTypeCase.number == ProfileAvatar.AVATAR_COLOR_HEX_FIELD_NUMBER) {
+        usedColors.add(it.profile.avatar.avatarColorHex)
```

Comment:
Will this work correctly? Here & elsewhere, I'm noticing a mixture of hex colors & color resource IDs. I think we should be storing the actual RGB value in the proto (since resource IDs may change between subsequent loads of the app), which means we should probably be converting all color resource IDs into hex upon load. It seems like we're interchanging these all over the place.

See also my comment on the proto file.

Reply:
I changed the implementation to now store the color hex as a string.